data/org.mate.pluma.gschema.xml.in

```diff
@@ -223,23 +223,23 @@
     </key>
     <key name="enable-space-drawer-newline" type="b">
       <default>false</default>
-      <summary>Draw newline</summary>
-      <description>Whether pluma should draw newlines in the editor window.</description>
+      <summary>Show newline</summary>
+      <description>Whether pluma should show newlines in the editor window.</description>
```

Comment:
typo? `"new lines"` instead `"newlines"`

Reply:
`"newline"` is the name of the control character - so we have these options to consider:
* Show newline
* Show newlines
* Show newline characters
* Show new lines

We can also do something like:
* Show line breaks
* Show line endings
* Show end-of-line characters

What do you think @mate-desktop/core-team ?

Reply:
you are right

https://en.wikipedia.org/wiki/Newline

you can forget my comment :)